larpandora/LArPandoraInterface/LArPandoraGeometry.cxx

```diff
@@ -241,84 +260,117 @@ void LArPandoraGeometry::LoadGeometry(LArDriftVolumeList &driftVolumeList)
             const float wireAngleU(0.5f * M_PI - theGeometry->WireAngleToVertical(targetViewU, itpc1, icstat));
             const float wireAngleV(0.5f * M_PI - theGeometry->WireAngleToVertical(targetViewV, itpc1, icstat));
             const float wireAngleW((nWirePlanes < 3) ? std::numeric_limits<float>::epsilon() : (useYPlane) ? (std::fabs(0.5f * M_PI - theGeometry->WireAngleToVertical(geo::kY, itpc1, icstat))) :
-                (0.5f * M_PI - theGeometry->WireAngleToVertical(geo::kW, itpc1, icstat)));
-
+                                   (0.5f * M_PI - theGeometry->WireAngleToVertical(geo::kW, itpc1, icstat)));
+            
             double localCoord1[3] = {0., 0., 0.};
             double worldCoord1[3] = {0., 0., 0.};
             theTpc1.LocalToWorld(localCoord1, worldCoord1);
-
+            
             const double min1(worldCoord1[0] - 0.5 * theTpc1.ActiveHalfWidth());
             const double max1(worldCoord1[0] + 0.5 * theTpc1.ActiveHalfWidth());
-
+            
             float driftMinX(worldCoord1[0] - theTpc1.ActiveHalfWidth());
             float driftMaxX(worldCoord1[0] + theTpc1.ActiveHalfWidth());
             float driftMinY(worldCoord1[1] - theTpc1.ActiveHalfHeight());
             float driftMaxY(worldCoord1[1] + theTpc1.ActiveHalfHeight());
             float driftMinZ(worldCoord1[2] - 0.5f * theTpc1.ActiveLength());
             float driftMaxZ(worldCoord1[2] + 0.5f * theTpc1.ActiveLength());
-
+            
             const bool isPositiveDrift(theTpc1.DriftDirection() == geo::kPosX);
-
+            
             UIntSet tpcList;
             tpcList.insert(itpc1);
-
+            
+            if (isDualPhase)
+            {
+                LArDaughterDriftVolumeList tpcVolumeList;
+                tpcVolumeList.push_back(LArDaughterDriftVolume(icstat, itpc1));
+                
+                driftVolumeList.push_back(LArDriftVolume(driftVolumeList.size(), isPositiveDrift,
+                                                         wirePitchU, wirePitchV, wirePitchW, wireAngleU, wireAngleV, wireAngleW,
+                                                         0.5f * (driftMaxX + driftMinX), 0.5f * (driftMaxY + driftMinY), 0.5f * (driftMaxZ + driftMinZ),
+                                                         (driftMaxX - driftMinX), (driftMaxY - driftMinY), (driftMaxZ - driftMinZ),
+                                                         (wirePitchU + wirePitchV + wirePitchW + 0.1f), tpcVolumeList));
```

Comment:
What's the `0.1f` here and might it need to be configurable? You could make a local const variable with a handy name to make this clearer.